package.json

```diff
@@ -12,8 +12,7 @@
     ]
   },
   "scripts": {
-    "test": "npm run lint && npm run prepublish",
-    "test-unit": "browserify test/*.test.js | tap-closer | smokestack | tap-status",
+    "test": "tape -r ./test/mock-browser.js -r babel-register test/*.test.js",
```

Comment:
Yes. The test-unit used to be the real tests but they couldn't run in circle so for circle we just tested if we could build. I don't think we need to test this anymore, but if you have concerns about not testing the build in circle (its implicitly tested by publishing) I'm all ears.